tools/bro-to-vast/bro-to-vast.cpp

```diff
@@ -167,11 +168,20 @@ class bro_writer : public vast::format::writer {
   bro_writer(broker::endpoint& endpoint, std::string query_id)
     : endpoint_{&endpoint},
       query_id_{std::move(query_id)} {
-    // nop
+    auto& cfg = endpoint.system().config();
+    show_progress_ = caf::get_or(cfg, "show-progress", false);
+  }
+
+  ~bro_writer() {
+    if (show_progress_ && num_results_ > 0)
+      std::cerr << std::endl;
+    VAST_INFO_ANON("query", query_id_, "had", num_results_, "result(s)");
   }
 
   caf::expected<void> write(const vast::event& x) override {
-    std::cerr << '.';
+    ++num_results_;
+    if (show_progress_)
+      std::cerr << '.';
```

Comment:
The dots are buffered, so the progress indication won't print anything until `std::endl` gets inserted into the stream: `std::cerr << '.' << std::flush;`